### DISCUSSION TITLE
Please bump web to 2023.9.1 Ukraine anthem FIX

### DISCUSSION THREAD

**Discussion by reysonk** [dani-garcia/vaultwarden#3933] [General]
We need an improved web version  to 2023.9.1 , since the problem would have been fixed by bitwarden
https://github.com/bitwarden/clients/issues/5734 "SweetAlert2 loads Ukraine anthem "

+ up here https://github.com/dani-garcia/vaultwarden/issues/3917

**Comment by reysonk** [comment]
https://hub.docker.com/layers/bitwarden/web/2023.9.1/images/sha256-1ce5acd5fad56f207729e309f521a0265a01cda72177096ef8bfacb787a8f8cd?context=explore

**Comment by stefan0xC** [comment]
FYI. I've already created a PR https://github.com/dani-garcia/bw_web_builds/pull/141

However, we are missing some changes to make vaultwarden compatible with this release and also some further testing would be good as there were many changes between the web-vault releases.

Also a heads up: This project is run by volunteers in their spare time so it could still take a while before this PR is merged and we have made the necessary changes to vaultwarden. If you can't wait, you can build the web-vault yourself with the patch file.
